include/i2c.h

```diff
@@ -56,11 +56,13 @@
  */
 typedef enum i2c_transfer_mode {
 	/** Address every device connected */
-	i2c_general_call =	0x01,
+	i2c_general_call = 0x01,
 	/** Send multiple start conditions */
-	i2c_repeated_start =	0x02,
+	i2c_repeated_start = 0x02,
 	/** Use 10-bit address scheme */
-	i2c_10_bit_transfer =	0x04
+	i2c_10_bit_transfer = 0x04,
```

Comment:
I'd use uppercase letters for enum's constants. It's easier to differeniate between them and other variables.

https://github.com/analogdevicesinc/no-OS/blob/b1cfb291db2e735fe61ad464ed573f475790d5ac/include/spi.h#L64-L73

https://github.com/analogdevicesinc/no-OS/blob/b1cfb291db2e735fe61ad464ed573f475790d5ac/drivers/adc/ad717x/ad717x.h#L56-L60

Reply:
True, I know something was wrong with the names but I was missing what was :)